.env.example

```diff
@@ -37,3 +37,4 @@ WEB_SOCKET_PORT=
 CONNECTION_LIMIT=
 WS_MAX_CONNECTION_TTL=
 WS_CONNECTION_LIMIT_PER_IP=
+WS_SUBSCRIPTION_LIMIT=
```

Comment:
Description for this ENV property should be included on [Docs](https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/docs/configuration.md)